300_Aggregations/75_sigterms.asciidoc

```diff
@@ -16,7 +16,7 @@ PUT /_snapshot/sigterms <1>
 {
     "type": "url",
     "settings": {
-        "url": "http://download.elasticsearch.org/definitiveguide/sigterms_demo/"
+        "url": "http://download.elastic.co/definitiveguide/sigterms_demo/"
```

Comment:
Pinged @drewr about this. Going to assume that we'll get it fixed on the site side.